### DISCUSSION TITLE
interesting intergration on bingai

### DISCUSSION THREAD

**Discussion by eu09** [waylaidwanderer/node-chatgpt-api#57] [Help] [answered]
can u give some sample how to work it ?

**Comment by waylaidwanderer** [comment, marked as answer]
Check out https://github.com/waylaidwanderer/node-chatgpt-api/blob/main/demos/use-bing-client.js - keep in mind that you need an account that already has access to it (i.e. not waitlisted).

**Reply by eu09** [reply]
ok , so I still need a browser to capture the cookie and paste as a userToken. A bit old school .  But still 
 looking forward to browserless support .
